DocumentsFromSnapshotMigration/src/main/java/org/opensearch/migrations/RfsMigrateDocuments.java

```diff
@@ -162,6 +163,11 @@ public static class Args {
             description = "The absolute path to the directory where we'll put the Lucene docs")
         public String luceneDir;
 
+        @Parameter(required = false,
+            names = { "--clean-local-dirs", "--cleanLocalDirs" },
+            description = "Optional. If enabled, deletes s3LocalDir and luceneDir before running. Default: false")
+        public boolean cleanLocalDirs = false;
```

Comment:
Is there a good reason that this should stay false as the default?  It seems strange to not clean up after yourself.